docs/source/build.rst

```diff
@@ -43,24 +43,26 @@ C/C++ Package
 
 .. code-block:: bash
 
-   conda install -c rapidsai -c conda-forge libcuvs cuda-version=12.9
+   conda install -c rapidsai -c conda-forge libcuvs cuda-version=13.0
 
 Python Package
 ~~~~~~~~~~~~~~
 
 .. code-block:: bash
 
-   conda install -c rapidsai -c conda-forge cuvs cuda-version=12.9
+   conda install -c rapidsai -c conda-forge cuvs cuda-version=13.0
```

Comment:
Can we add "For CUDA 12" and "For CUDA 13" commands here as well to have uniformity with the Pip commands section?